### PR TITLE
Release notes 113 oss

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,41 @@
 
 This document is automatically generated at {{gitbook.time}}
 
+## 1.1.3 (WIP)
+
+- Updated contributorlist, with 2 new co-authors.
```
Comment (TheDauntless) [inline CHANGELOG.md]: contributorlist in one word?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,41 @@
...
+- Translated various parts into Japanese.
+- A large restructuring of the general testing, platform specific testing and reverse-engineering chapters.
+- Updated description of many tools: Adb, Angr, APK axtractor, Apkx, Burp Suite, Drozer, ClassDump(Z/etc), Clutch, Drozer, Frida, Hopper, Ghidra, IDB, Ipa Installer, iFunBox, iOS-deploy, KeychainDumper, Mobile-Security-Framework, Nathan, Needle, Objection, Magisk, PassionFruit, Radare 2, Tableplus, SOcket CAT, Xposed, and others.
+- Updated most of the iOS hacking/verification techniques using iOS 12 or 11 as a base instead of iOS 9/10.
+- Removed tools which were no longer updated, such as introspy-Android and AndBug.
+- Added missing MASVS references from version 1.1.4: v1.X, V3.5, V5.6, V6.2-V6.5, V8.2-V8.6.
+- Rewrote device-binding explanation and testcases for Android.
+- Added parts on testing unmanaged code: things to check for at unsafe Objective-C, Parcelling in Java, and the usage of C/C++.
```
Comment (TheDauntless) [inline CHANGELOG.md]:
```suggestion
- Added parts on testing unmanaged code in Objective-C, Java, and C/C++.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,41 @@
...
+- Updated many cryptography related parts.
+- Added testaces for upgrade-mechanism verification for apps.
+- Updated Readme, Code of Conduct, Contribution guidelines, verification, funding link, and generation scripts.
+- Added ISBN as the book is now available at Hulu.
+- Added various fixes for the .epub format.
+- Added testcases on Android and iOS backup verification.
+- Improved key-attestation related explanation for Android.
+- Restructured OWASP Mobile Wiki.
+- Removed Yahoo Weather app and simplified reference on using SQL injection.
+- iOS app sideloading is better explained with the various methods available.
```
Comment (TheDauntless) [inline CHANGELOG.md]:
```suggestion
- Improve explanation for iOS app sideloading to include various available methods.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,41 @@
...
+- Added explanation on using ADB and device shell for Android.
+- Added explanation on using device shell for iOS.
+- Provided comparison for using emulators/simulators and real devices for iOS/Android.
+- Fixed Uncrackable Level 3 for Android.
+- Improved explanation on how to exfiltrate data and apps on iOS 12 and Android 8.
+- Improved/updated explanation on SSL-pinning.
+- Added list of adopters of the MASVS/MSTG.
+- Updated English, Japanese, French and Spanish checklists to be compatible with MSTG 1.1.2.
+- Added a small write-up on Adiantum for Google.
+- Added MSTG-ID to the paragraphs create a better link between MSTG paragraphs and MASVS requirements.
```
Comment (TheDauntless) [inline CHANGELOG.md]:
```suggestion
- Added MSTG-ID to the paragraphs to create a link between MSTG paragraphs and MASVS requirements.
```

**Comment by OMTGreleaser** [comment]
Results for commit 39661f5fa228fa5fc3eb6121635858c973ea1bab: Broken link result: [!] Error(s) found by the Links validator: 5 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit d9987c507dcf5125b8bf5eac361e72582bb22bf0: Broken link result: [!] Error(s) found by the Links validator: 5 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**